framework/src/Volo.Abp.BlazoriseUI/Components/PageHeader.razor

```diff
@@ -0,0 +1,35 @@
+﻿<Row Class="entry-row">
+    <Column ColumnSize="ColumnSize.IsAuto">
+        <h1 class="content-header-title">@Title</h1>
```

Comment:
You can use `Heading` component instead of `<h1>` element

Reply:
Yes. I used the `Heading `component but had to change back to `<h1>` element. Heading component adds "h1" class while rendering  the `<h1>` element and it breaks the lepton-theme design.

Reply:
I will probably need to fix that in Blazorise